### DISCUSSION TITLE
Initial setup commands: UI Language selection, set clock

### DISCUSSION THREAD

**Discussion by ertdredge** [gopro/labs#807] [Q&A]
Hello!  I have a HERO8 Black with a broken rear screen which is stuck waiting for me to set the UI language and system clock.  I was hoping I could use GoPro labs to set the camera up and control it enough for my needs, but I'm having some challenges.

I've been able to set the clock via a QR command -- I can see in qrlog.txt that it worked, but whenever it gets back to Labs:Initialize it resets to 2020-01-01.  And while the oL0 command is being read in I gather that is for setting the voice control language and not the main UI language.  I _think_ what is happening is that I'm still being asked via the dead rear screen to choose the language.

I cannot easily record videos -- the record button does nothing, and while I can start recording with a QR code the only way I can stop it is by powering down the camera one way or another which seems to corrupt the recording as often as not.

Any ideas about ways I might be able to use GoPro Labs to configure these basic options that the camera is waiting for, or at least getting it to a place where the record button isn't ignored?

**Comment by dnewman-gpsw** [comment]
If the screen is attached but not working, the camera will expect touch input.  Labs does support screenless operations, but you can try scanning !MNAKE=1 to make behave if the screen is removed.  This may work, although is more optimized around HERO10 Labs developed.

**Reply by ertdredge** [reply]
Thank you for the suggestion, @dnewman-gpsw!  Unfortunately, I didn't observe any behavior changes on the camera after scanning !MNAKE=1

**Reply by dnewman-gpsw** [reply]
Thought that might be.   Next thing to try is !MBYPS=1  to bypass popups on boot.  Gain this is feature designed for later camera, but some may have been back-ported to HERO8.

**Reply by ertdredge** [reply]
Ooh, that sounded like exactly what I wanted, but sadly no discernible effect on this HERO8 😢

**Reply by y-maskin** [reply]
!MBYPS=1 helped me to bypass the language screen on a HERO 9 but the screen still doesn't react to any touch. Are there any codes I can try to reset the screen connection/any tests that could help me validate whether it's software or hardware?
